browser/ephemeral_storage/ephemeral_storage_tab_helper.cc

```diff
@@ -104,9 +109,18 @@ void EphemeralStorageTabHelper::CreateEphemeralStorageAreasForDomainAndURL(
 
   std::string session_partition_id = StringToSessionStorageId(
       content::GetSessionStorageNamespaceId(web_contents()),
-      "/ephemeral-session-storage");
-  session_storage_namespace_ =
-      content::CreateSessionStorageNamespace(partition, session_partition_id);
+      kSessionStorageSuffix);
+
+  auto* rfh = web_contents()->GetOpener();
+  session_storage_namespace_ = content::CreateSessionStorageNamespace(
+      partition, session_partition_id,
+      // clone the namespace if there is an opener
```

Comment:
really a nit, but the style guide wants us to use proper punctuation, i.e. start sentenses with a capital and end with a dot.

https://google.github.io/styleguide/cppguide.html#Punctuation,_Spelling_and_Grammar